src/components/transaction-title.tsx

```diff
@@ -70,21 +71,14 @@ export const getTxTitle = (transaction: Transaction | MempoolTransaction) => {
 };
 
 export const TransactionTitle = ({ contractName, tx, ...rest }: TitleProps & StackProps) => {
-  let txStatus: TxStatus;
-  if (tx.tx_status === 'success' && !!tx.is_unanchored) {
-    txStatus = 'success_microblock';
-  } else if (tx.tx_status === 'success' && !tx.is_unanchored) {
-    txStatus = 'success_anchor_block';
-  } else {
-    txStatus = tx.tx_status;
-  }
+  const txStatus = useTransactionStatus(tx);
```

Comment:
If you don't want to repeat the `useMemo` wrapper of the function, you _could_ make a hook the contains just the 
`useMemo(() => getTransactionStatus(tx), [tx])` called `useMemoizedTxStatus` or something.

But personally I don't think this abstraction is super important.